payload-receiver-service/tests/integration/smoke_tests.py

```diff
@@ -0,0 +1,288 @@
+import asyncio
+import json
+import subprocess
+import time
+from queue import Queue
+from subprocess import PIPE, Popen
+from threading import Thread
+
+import requests
+from faker import Faker
+from nats.aio.client import Client as NATS
+from nats.aio.errors import ErrConnectionClosed, ErrNoServers, ErrTimeout
+from opni_nats import NatsWrapper
+
+kube_fname = "kube.yaml"
+nw = NatsWrapper()
+fake = Faker()
+port_forward_command = "kubectl --kubeconfig kube.yaml -n opni-demo port-forward "
```

Comment:
For the newest Opni release, it uses the opni namespace instead on opni-demo namespace.

Reply:
The port forwarding commands for payload receiver service and Nats are repeated throughout the code. Declare the port forwarding commands for the two services here and simply use those variables in the function.
Example:
```
port_forward_command_nats = "kubectl --kubeconfig kube.yaml -n opni port-forward svc/cluster-nats-client 4222:4222"
port_forward_command_prs = "kubectl --kubeconfig kube.yaml -n opni port-forward svc/opni-svc-payload-receiver 8080:80"
```